src/App/Hangouts/Commands.elm

```diff
@@ -1,34 +1,45 @@
 module App.Hangouts.Commands exposing (..)
 
 import Http as H
-import HttpBuilder exposing (withExpect)
-import Json.Decode as Decode exposing (Decoder, succeed, string, int)
+import HttpBuilder exposing (withExpect, withHeader, withJsonBody, send)
+import Json.Decode as Decode exposing (Decoder, succeed, string, int, maybe)
 import Json.Decode.Extra exposing ((|:))
+import Json.Encode as Encode exposing (object, string)
 import Common.Utils.Http as Http
+import Common.Utils exposing (hangoutsUrl, shuffleUrl)
 import App.Decoders.Common exposing (..)
 import App.Hangouts.Messages exposing (Msg(..))
 import App.Hangouts.Models exposing (Hangout, Group)
 import App.Auth.Models exposing (Token, User)
 import App.Auth.Commands exposing (userDecoder)
 
 
-hangoutsUrl : String
-hangoutsUrl =
-    baseUrl ++ "/hangouts/"
-
-
 getHangouts : Token -> Cmd Msg
 getHangouts =
     HttpBuilder.send OnFetchHangouts
         << withExpect (H.expectJson <| decodeCollection hangoutDecoder)
         << Http.get hangoutsUrl
 
 
+shuffleHangouts : Token -> Cmd Msg
+shuffleHangouts token =
+    Http.post shuffleUrl token
+        |> withJsonBody hangoutsPayload
+        |> withExpect (H.expectJson hangoutDecoder)
+        |> HttpBuilder.send OnShuffleHangouts
+
+
+hangoutsPayload : Encode.Value
+hangoutsPayload =
+    object <|
```

Comment:
**Suggestion** - Since we have `Encode.string` in the following line, maybe fully qualify this too and we can remove the `exposing (object, string)` from the import?